### DISCUSSION TITLE
How often does Copilot gets trained on public repositories?

### DISCUSSION THREAD

**Discussion by AathmanT** [community/community#19032] [Copilot Conversations] [answered]
Is it possible to know how often does Copilot gets trained on public repositories, and the last time the model got updated?

**Comment by NeuralAIM** [comment, marked as answer]
Perhaps this is the answer to your question.

![image](https://user-images.githubusercontent.com/75082388/174886549-e3aaf153-d1d1-4f8d-8f21-a6a2ef160e90.png)

**Reply by AathmanT** [reply]
Thank you for pointing out the last training date. I would also like to know about the training frequency as well.

**Reply by NeuralAIM** [reply]
> Thank you for pointing out the last training date. I would also like to know about the training frequency as well.

I can assume that the neural network is not trained often :)

![image](https://user-images.githubusercontent.com/75082388/175070988-7898154b-4cbd-4d07-9406-6c6d07095b48.png)

**Reply by AathmanT** [reply]
Agreed. It looks like it.
